client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -989,130 +701,3 @@ export default connect(
 		requestSite,
 	}
 )( localize( CheckoutThankYou ) );
-
-/**
- * Retrieves the component (and any corresponding data) that should be displayed according to the type of purchase
- * just performed by the user.
- */
-function PurchaseDetailsWrapper(
```

Comment:
Similarly, this function should be kept, but with all conditions except for `hasFailedPurchases` removed.